content/DC_resistivity/DC_resistivity_surveys.rst

```diff
@@ -65,4 +66,6 @@ Soils and rocks are composed mostly of silicate minerals, which are essentially
 	- temperature and phase of pore fluid;
 	- amount and composition of colloids (clay content).
 
-Detailed discussion of geologic factors affecting this important physical property are provided in a separate location.
\ No newline at end of file
+Detailed discussion of geologic factors affecting this important physical property are provided in a separate location.
```

Comment:
Hi Lindsey,

                I’m at home. Phone me anytime that it is convenient. 604-868-3448

Doug

 

From: Lindsey [mailto:notifications@github.com] 
Sent: September-05-15 11:52 AM
To: ubcgif/geoscixyz
Cc: dougoldenburg
Subject: Re: [geoscixyz] Thibaut (#6)

 

In content/DC_resistivity/DC_resistivity_surveys.rst <https://github.com/ubcgif/geoscixyz/pull/6#discussion_r38813667> :

> \ No newline at end of file
> +Detailed discussion of geologic factors affecting this important physical property are provided in a separate location.

could you add a link here? 

—
Reply to this email directly or view it on GitHub <https://github.com/ubcgif/geoscixyz/pull/6/files#r38813667> .  <https://github.com/notifications/beacon/AIdCvGUOuJgKdKPlKMJ8BpIQDBfwad0cks5ouzFigaJpZM4F2H9Q.gif>